toast-tk-dao/src/main/java/io/toast/tk/dao/domain/impl/repository/ProjectImpl.java

```diff
@@ -0,0 +1,27 @@
+package io.toast.tk.dao.domain.impl.repository;
+
+import org.bson.types.ObjectId;
+
+import com.github.jmkgreen.morphia.annotations.Entity;
+import com.github.jmkgreen.morphia.annotations.Id;
+import com.github.jmkgreen.morphia.annotations.Index;
+import com.github.jmkgreen.morphia.annotations.Indexes;
+
+import io.toast.tk.dao.domain.impl.common.BasicTaggableMongoBean;
+import io.toast.tk.dao.domain.impl.test.block.IProject;
+
+@Entity(value = "projects")
+@Indexes({ @Index(value = "name") })
+public class ProjectImpl extends BasicTaggableMongoBean implements IProject {
+
+	@Id
+	ObjectId id = new ObjectId();
+	
+	/**
+	 * serialization
+	 */
+	public ProjectImpl(){

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AS1186)